### ISSUE TITLE
Research Bluebeam enterprise license for ROW

### ISSUE THREAD

**Issue by moorerst** [cityofaustin/atd-data-tech#2144]
From Ryan M.: Hello, with the increase of remote work, I think we should look at some Bluebeam enterprise licenses. I understand from DSD they are purchasing ~300, which are concurrent user licenses and come with automatic upgrades. Ours are all currently installed per machine and locked into the purchased version. I've heard from folks in TED they would find it useful. I don't think we need 300 though. Just a department-wide thought since it looks like we will be in this situation for a couple of months potentially.

**Comment by moorerst** [comment]
Per John coordinate with Scott Reedy:
Per Scott...
The problem with that is we lose control over our licenses and cannot guarantee our users will be able to obtain a license. It is first come first serve with all departments as it is set up now. 

Stephanie can inquire if an individual dept can get their own “enterprise” license or is it just one license per entity, ie City of Austin, just like the adobe license setup. 

Scott Reedy

**Comment by moorerst** [comment]
Search ServiceNow for DSD Purchase Request for Bluebeam licenses. Located PR0010882 for 300 concurrent licenses for DSD. Vendor contact is Michael Greiff mgreiff@bluebeam.com.
Provided info via email (Scott, John, Ryan, Peggy) - I located the DSD Purchase Request for 300 Bluebeam “concurrent (open)” licenses @ 1,119.61 per license.
I will reach out to the vendor to see what license options are available.

Reply for Scott...
Stephanie, 
Let’s hold off please until DSD gets the current problem under control.
I fear your contacting them will cause more confusion on an already very confusing situation.

**Comment by amenity** [comment]
@moorerst - Thanks for your legwork on this! Will you assign an estimate to this issue that includes the work that you did last week? I'm going to put this in the "Blocked" pipeline to remind us to check back in with DSD and/or Scott. 
If you haven't already, will you circle back with Ryan M. to let him know where the request stands?🙏

**Comment by moorerst** [comment]
Per SR no additional actions needed on this request
The problem has been corrected with the bluebeam licenses. I learned that we can have a pool of licenses as described by Ryan. However, it would be for all of ATD which would encompass TDS and Engineering as well as ROW. 

I don’t think there is anything left to find out. I would need to supply the licenses, and double check with the vendor that they don’t roll everyone’s licenses throughout the city into our request ( as they did with DSD). I would also need to let DSD, PWD and CTM know that we are doing  this just as a warning. The problem lies with the resellers, if they don’t listen to what exactly we are asking.
